src/Admin/views/settings.php

```diff
@@ -69,7 +80,7 @@
                 <th scope="row">Notification Email</th>
                 <td>
                     <input type="email" name="notification_email" value="<?php echo esc_attr($notification_email); ?>" />
-                    <p class="description">Enter the email address where notifications will be sent.</p>
+                    <p class="description">Enter one or more email addresses separated by commas where notifications will be sent.</p>
```

Comment:
add `_()` or `_e()` function for translatable string

Reply:
Updated